crates/client/assets/schema.sdl

```diff
@@ -1128,6 +1128,10 @@ type Subscription {
 	Submits transaction to the `TxPool` and await either confirmation or failure.
 	"""
 	submitAndAwait(tx: HexString!): TransactionStatus!
+	"""
+	Submits transaction to the `TxPool` and throws failure immediately or returns `SubmittedStatus`, then waits for confirmation or failure.
+	"""
+	submitAndAwaitStatus(tx: HexString!): TransactionStatus!
```

Comment:
We should test how this works with the load balancer config, as I think long blocking http requests are automatically times out.

Reply:
I don't see much difference with `submitAndAwait`, if it works, then a new method should work as well